### ISSUE TITLE
doc about groovy script for automation

### ISSUE THREAD

**Issue by dktodorov** [KostyaSha/yet-another-docker-plugin#279]
Hello,

Can you update the script for automation, because it is outdated and launch_ssh is not working? 

Maybe in the past constructor for DockerComputerSSHLauncher was able to accept SSHConnector, but now it looks like that it should be used DockerSSHConnector.

**Comment by KostyaSha** [comment]
Sorry, what script?

**Comment by dktodorov** [comment]
https://github.com/KostyaSha/yet-another-docker-plugin/blob/master/docs/script-console-scripts/configure-yadocker-cloud.groovy

**Comment by KostyaSha** [comment]
Yes, sure. I think now jcasc could be added also!

**Comment by serbaniuliuscezar** [comment]
Would also be useful for me :)

**Comment by 33man** [comment]
I just updated it to match new constructor of DockerSSHConnector => https://github.com/33man/yet-another-docker-plugin/blob/master/docs/script-console-scripts/configure-yadocker-cloud.groovy

**Comment by dktodorov** [comment]
@33man Sorry, but I try to use your code, but I have some issues, can you suggest where is my mistake?

`import com.github.kostyasha.yad.commons.DockerSSHConnector
import com.github.kostyasha.yad.launcher.DockerComputerSSHLauncher
import hudson.plugins.sshslaves.verifiers.NonVerifyingKeyVerificationStrategy


DockerSSHConnector sshConnector = new DockerSSHConnector(
  22,
  null,
  'yad-test',
  '',
  '',
  null,
  '',
  '',
  120,
  10,
  10,
  new NonVerifyingKeyVerificationStrategy()
)
launcher =  new DockerComputerSSHLauncher(sshConnector)

groovy.lang.GroovyRuntimeException: Could not find matching constructor for: com.github.kostyasha.yad.commons.DockerSSHConnector(java.lang.Integer, null, java.lang.String, java.lang.String, java.lang.String, null, java.lang.String, java.lang.String, java.lang.Integer, java.lang.Integer, java.lang.Integer, hudson.plugins.sshslaves.verifiers.NonVerifyingKeyVerificationStrategy)
`

**Comment by 33man** [comment]
Hello, which version of the plugin are you using ? I tested with latest 0.1.5

**Comment by dktodorov** [comment]
I am using 0.1.4, I will try to update tomorrow.
Thank you

**Comment by 33man** [comment]
Yep the 0.1.5 added new argument to the constructor : https://github.com/KostyaSha/yet-another-docker-plugin/compare/0.1.4...0.1.5#diff-18e1cc7bbe4c4d6477b7e68f87e8bdbf

**Comment by dktodorov** [comment]
@33man thanks, you were right.
After update it is working.
